examples/notebooks/PySpark with Bigtable Time-Series Example.ipynb

```diff
@@ -0,0 +1,489 @@
iuhu+/vrr0qtXLxPrSbzngQceaGKibLtlWbWjeZ+jD/TCvR+pUAe9V4UJLttnYmO7du1aDPzYfdEsMzMzK+WZi+aYvqzXgNeA10B8XIyc0jNZ0lJjJE+HLaQRTVQzyuqN+fLuL5ny+FebZNzc3H+MomKjGXyV56zweca8j4WDH/igHHrooWb/LbfcEtpl3dOefvppE2x//PHHmx/0bt26ybHHHitstz9sQ4YMCbl22ZnPUEP6hYGTbY9jXHHFFbL33nuHilCH/fTDlV9//VVuuOEG2WuvvUyg83777WfW//jjD7eYCci27f/0009y++23mx85BgdTp04tVra0FdplUBGNjBgxwvQJ3eIrTtA/7hQ2boO2uL70j9lHK8yesw3LkyuRnrNbh4Hd//73P9Meg69OnToZHbuDJbf8vHnz5L777jP65noyUCOegmtaEhjifDgvzo/z5Hy5NiNHjnSbLvU7cUv2Ol122WWllmVnJLr49NNPQ23i+uMKAc/0lWNyz9rnDH299NJL5vyJnTrooIOkXbt2cuaZZ8r48ePdJuSaa64x9dnHwJUlM+7ci7179zbxdlQYNGiQORb6ZNBLvMD3339fYluADWLLmI2mj3zuueceQyxQrFIJKwwu33vvPbnwwgvN9d5///3N+XBdIU2IVCK5f2mL633CCSeEmrUuQ8S0lCTlee/QFrrkeu22225mgM89x/05evToYofi2eHaomsr7vXCYoV+mOnnGtPOzTffHBag/vzzz+Z6UBagwjV+4YUXZMaMGaH7q7xWG/pGPM4FF1xguym4w0b73qSy+z5r3LhxqD37BRevl19+WXArdq0/9rmL5H1u22I5e/Zso0P0YfXCfY/1DfDuinsMYpR4RtC7+y50y9vvkbwXhg0bFroOWMFc4Vm2x54+fbq7y7gDso/7l+eeeElblnPjfc+6/U3hubT7XYunbXTixImG9AOrI88cuiB+tCKA2bbtl14DXgNeA5FooHm9ODnzoCQDfABBSKwCoXi1CM1clifPqlvc6z/q5O6aqu9pFL+tg1WZrbImfn4EgoxnMBex3wIalMk6AqDgR9MVXC0sC9QTTzxhZh5teQYvZ5xxhltcfvvtt1B7/IjiwmHL24Ks47phhcEvLD2urF69Wj7//HPzYeDLTB7COdn2Hn74YYEdKhq54447zAwt9R566CHzow2IKEvC9ZE6AA8+WGOuvfZa04ztn9sm21xLQrj2Sjpnt50HHnhgK8ILXGD4MEi2M8/UYUaWwSTtukIQOR9iFV555RVJTEwM7eb6B+8Bt18Mdph9Lk0AA5a0AcAVBCvBupHqgkH5u+++KwxMGAgdd9xxBlTQ3pNPPhmyrvz73/82g1AGKgzecQ8KCoMgXKoYFDGwQQCEXCfILxhMu9YagAakI9w/wTg27iVAnqt/2xb7qOfe77SLmxTnzYAQF53ShOthg7ltOdrj8+OPP5pnDCKU0iScjikf7v4N3i+UQy+uVYhtrpTnvUOczGOPPeY2Y75zbxJz9Pzzz0v//v3NNts+97QVq2OuF+8h93pRhveG1bWtA7BywQnbObfHH39cAEZ8R6KxfpgKgX8uKyfvBp5D23Yk702a23fffUOgG/c3YrKSkpKKHSkcI549TiTvc9sYzwOAPyjc94Asnrm33nor9Htij/HVV18Vu7eD9d31SN8LTJjZ9rlegCsrtGH3MSHDpIIVnnP2MSkBkyLWUluWCRGsjHbd1rHrwckgSEh4bu1+yqOLp556ygAg+6637fil14DXgNfAttLAgbslahxQjHzwW6ZafwokPrbQGsQSh5Yf/84xlqB+6hZ3yB6JUj25arrFVelEqPzIIbBRAT4YuDZq1MhsY50BXs2aNc2MLRthoGPm3RV3Jvywww4TQBOzn1YIPKUMs3wIA0QLfjgWbD3MVGPlsIM69gcHN7Yug1cGiAzMmzVrZtos7R+uNO4A///+7/+EwVVpgjXA9hGff4AT58AAjplmBMYhBlwAXPY9+OCDoSaxhLHNBpVX5Jxh++MH/rnnnpNPPvlELr/88tBxYIOyQdf8oJ9zzjkh8IMFghlRqIT79etn6vzyyy/GgmYboP9WN8zIc370m/O1QpvBmVe7jyXXzlp8aINBkzsYdMvyPRpdMKhxB8yAEUAOA2DuTYRBiwVoDKQt+IEBivPjvqOelXBWQDvoARwywHOteYAf4jGohz45nhXATFAAEwCVc88918yOA9wsSGU77HmlCa5NFvwcfPDB5hw4p3vvvddUY1B20UUXlTorHc39S6PcFwz0rKAv7gN7b9jtFVkykYHVBQEkA9C4F7inecaQ0tgaTYGif1wv3g+8r9DX9ddfH2qDCRlLIc8EhAt+eF64J3jXYL2oTCZNrD5WeF6jfW9S1wU3WFawOGLV4l08a9Ys23yJy0je57ayq2v0CHji2cWagvCut+3ZOiy5h2EH5H3HM4ibYjiJ5r1Qr149c0/QjmsJxN3UXkv2cW2tMLHHfY7YCQ27zy55d3Ef23c2vy+s8wmy+nEvQm/Oe5Z7hOfXCu/6igJk25Zfeg14DXgNFKgKVqTny8yluYbgQIc1W8kBHRLkxmNTpVubeOMKl1tk8MEtLkGtQes3F8iHI7PksS8zZMycHAOMtmpkB2+Ixy+5KguDHPzYEdzRGJxYthwY1Bjw4SLHwBBh1pR1hBk264vONgYyfOrUqWP284/BcNu2bUPrLjjiB9f+OOHywQD0pJNOMrNwzPYxY+8KM6SUCVq53DLhvjMwtzP5/NAB0nClK0lsrhv2Y2mwAxPOA+sR1NoMwgYOHCgnn3yyOT93Fp1zqaxzBiS++uqrIRpgdMDsJnqgD3/++ae5fgxWGJwgWDMYFFrBmsYMO2Ww0DGoZ2aZwSfCNWPWt3nz5madvkM7bMEWAwZ35tUU0n8M+hhkIAyKAKVl0RVHe/25P4jfAggBcOg/ViEE3WD9scIMMOeO4IZo87Uw2GGQz4dBFOWCM+v
```

Comment:
let's change these to something like `bigtable_project_id = "BIGTABLE_PROJECT_ID"` as discusses offline

Reply:
updated